src/components/settings/SignerAccountMFA/index.tsx

```diff
@@ -1,21 +1,21 @@
-import useMPC from '@/hooks/wallets/mpc/useMPC'
 import { Box, Typography } from '@mui/material'
-import { COREKIT_STATUS } from '@web3auth/mpc-core-kit'
 
 import { PasswordForm } from './PasswordForm'
+import useWallet from '@/hooks/wallets/useWallet'
+import { ONBOARD_MPC_MODULE_LABEL } from '@/services/mpc/module'
 
 const SignerAccountMFA = () => {
-  const mpcCoreKit = useMPC()
+  const wallet = useWallet()
 
-  if (mpcCoreKit?.status !== COREKIT_STATUS.LOGGED_IN) {
+  if (wallet?.label !== ONBOARD_MPC_MODULE_LABEL) {
```

Comment:
Nit: I think we are using this check quite frequently across the app.
Should we add a small helper `isSocialLogin(wallet)`?

Reply:
done